specifyweb/frontend/js_src/lib/components/DataModel/businessRuleDefs.ts

```diff
@@ -60,7 +60,7 @@ type MappedBusinessRuleDefs = {
 export const nonUniqueBusinessRuleDefs: MappedBusinessRuleDefs = {
   BorrowMaterial: {
     fieldChecks: {
-      quantityReturned: (
+      quantityreturned: (
```

Comment:
I have toyed around with using camelCase instead of lowercase for the fieldCheck definitions. 

The only issue is that we would have to convert the fieldnames coming from BackBone-which are in lowercase- _back_ to camelCase which is substantially more difficult than the other way around. 

To invoke the correct rule, we need to be able to use the BackBone field names (in lowecase) to pull the right field from fieldCheck definitions. Here is where the issue occurs in the code:

https://github.com/specify/specify7/pull/2965/files#diff-afde7250d8079dd81ae9657ed120cf46a1f5147a88ac908f3d20840d79d88ee2R369

Reply:
The .getField() and .getFields() methods on SpecifyModel class are perfect for converting lowercase names back to camel case

Also, keep in mind that we are planning to get rid of backbone (and it's lowercase fields too), thus don't contain your design by the limitations of a legacy soon to be removed system

Reply:
Great! `.getField()` sounds exactly like what I need then. Thank you!

Reply:
Fixed in 373d4ba